tests/DbStorage_1.py

```diff
@@ -86,7 +86,7 @@ def testWriteRead(self):
 
         db.query()
 
-        self.assertTrue(next(db))
+        self.assertTrue(db.next())
```

Comment:
if it's expected in our code that next(x) would be used instead of x.next() I think it is worth copying what you put in the checkin comment into a comment above this line so future developers & reviewers don't flag this and go thru the same process of discovery.